cmake/mender_mcu_sources.txt

```diff
@@ -161,7 +161,7 @@ if (CONFIG_MENDER_CLIENT_INVENTORY)
 endif()
 if (CONFIG_MENDER_ZEPHYR_IMAGE_UPDATE_MODULE)
     list(APPEND MENDER_MCU_SOURCES
-        "${MENDER_MCU_ROOT}/src/core/zephyr-image-update-module.c"
+        "${MENDER_MCU_ROOT}/src/platform/update_modules/zephyr/image/update_module.c"
```

Comment:
This is okey, but consider going a tiny but further and get [this one done](https://northerntech.atlassian.net/browse/MEN-7554).